src/tokens/TokenBalances.tsx

```diff
@@ -135,8 +122,19 @@ function TokenBalancesScreen({ navigation }: Props) {
   return (
     <>
       {shouldVisualizeNFTsInHomeAssetsPage && (
-        <Touchable testID={'NftViewerBanner'} borderless={true} onPress={onPressNFTsBanner}>
-          <View style={[styles.bannerContainer]}>
+        <Touchable
+          style={
+            // For larger fonts we need different marginTop for nft banner
+            PixelRatio.getFontScale() > 1.5
```

Comment:
Hmm, this seems weird to me, but I don't know how to do it in a different way.

Reply:
I agree it's strange, but it works 🤷